draft-ietf-quic-http.md

```diff
@@ -1193,6 +1198,10 @@ Push Stream Header {
 ~~~~~~~~~~
 {: #fig-push-stream-header title="Push Stream Header"}
 
+A client SHOULD NOT abort reading on a push stream prior to reading the Push ID,
```

Comment:
```suggestion
A client SHOULD NOT abort reading a push stream prior to reading the Push ID,
```

maybe?

Reply:
Other parts of the doc use the style "reading on", so I'll remove this suggestion.